<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LikeBackBar</class>
 <widget class="QWidget" name="LikeBackBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="m_likeButton">
     <property name="toolTip">
      <string>Click here to tell the developers about something you liked</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;Click on one of these icons to send your feedback to the developers of this application. You can disable the icons with the &quot;Show Feedback Icons&quot; checkbox in the Help menu.&lt;/html&gt;
      </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_dislikeButton">
     <property name="toolTip">
      <string>Click here to tell the developers about something you did not like</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;Click on one of these icons to send your feedback to the developers of this application. You can disable the icons with the &quot;Show Feedback Icons&quot; checkbox in the Help menu.&lt;/html&gt;
      </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_bugButton">
     <property name="toolTip">
      <string>Click here to tell the developers about a problem in the application</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;Click on one of these icons to send your feedback to the developers of this application. You can disable the icons with the &quot;Show Feedback Icons&quot; checkbox in the Help menu.&lt;/html&gt;
      </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_featureButton">
     <property name="toolTip">
      <string>Click here to tell the developers about new features you would like to have in this application</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;Click on one of these icons to send your feedback to the developers of this application. You can disable the icons with the &quot;Show Feedback Icons&quot; checkbox in the Help menu.&lt;/html&gt;
      </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_likeButton</sender>
   <signal>clicked()</signal>
   <receiver>LikeBackBar</receiver>
   <slot>likeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_dislikeButton</sender>
   <signal>clicked()</signal>
   <receiver>LikeBackBar</receiver>
   <slot>dislikeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_bugButton</sender>
   <signal>clicked()</signal>
   <receiver>LikeBackBar</receiver>
   <slot>bugClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_featureButton</sender>
   <signal>clicked()</signal>
   <receiver>LikeBackBar</receiver>
   <slot>featureClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>likeClicked()</slot>
  <slot>dislikeClicked()</slot>
  <slot>bugClicked()</slot>
  <slot>featureClicked()</slot>
 </slots>
</ui>
